test/utils/assert.js

```diff
@@ -1,15 +1,19 @@
 'use strict';
 
-var assert = require('assert');
-
+const assert = require('assert');
+const mwUtil = require('../../lib/mwUtil');
 
 /**
  * Asserts whether content type was as expected
  */
 function contentType(res, expected) {
-    var actual = res.headers['content-type'];
-    deepEqual(actual, expected,
-        'Expected content-type to be ' + expected + ', but was ' + actual);
+    const actual = res.headers['content-type'];
+    if (/^\/.+\/$/.test(expected)) {
```

Comment:
BTW you could also use `expected.source` to see if it's a regex. If it's not then it would return undefined.

> 'foo'.source
undefined

Reply:
Here it's never an instance of a `RegEx` cause it's a string read from the yaml, so testing for `/` in the beginning and the end is the only way